addons/logistics_wirecutter/functions/fnc_cutDownFence.sqf

```diff
@@ -10,52 +10,60 @@
  * None
  *
  * Example:
- * [player, berlinWall] call ace_logistics_wirecutter_fnc_cutDownFence
+ * [_player, _fence] call ace_logistics_wirecutter_fnc_cutDownFence
```

Comment:
example is mostly useful for copy-pasting to debug console so `[player, cursorObject] call ace_logistics_wirecutter_fnc_cutDownFence` would be better